opentelemetry-instrumentation/src/opentelemetry/instrumentation/auto_instrumentation/sitecustomize.py

```diff
@@ -32,24 +32,28 @@
 
 
 def _load_distros() -> BaseDistro:
-    for entry_point in iter_entry_points("opentelemetry_distro"):
-        try:
-            distro = entry_point.load()()
-            if not isinstance(distro, BaseDistro):
-                logger.debug(
-                    "%s is not an OpenTelemetry Distro. Skipping",
-                    entry_point.name,
+    entry_points = iter_entry_points("opentelemetry_distro")
+    try:
+        first_distro = next(entry_points)
+
+        more_distros = [ep.module_name for ep in entry_points]
+
+        if more_distros:
+            logger.error(
+                    "Multiple distros were found: (%s). Only one should be installed.",
+                    ",".join([first_distro.module_name] + more_distros),
                 )
-                continue
-            logger.debug(
-                "Distribution %s will be configured", entry_point.name
-            )
-            return distro
-        except Exception as exc:  # pylint: disable=broad-except
-            logger.exception(
-                "Distribution %s configuration failed", entry_point.name
-            )
-            raise exc
+            raise RuntimeError("Cannot Auto Instrument with multiple distros installed.")
+
+        return first_distro.load()()
+    except IndexError:
```

Comment:
this would be `StopIteration` I think.

Reply:
Right, thank you! I changed it and configured that's what we wanted in my tests 😄 

```
opentelemetry-instrument python3 auto-instrumentation/flask/application.py
Initializing Auto Instrumentation without using a distro.
 * Serving Flask app "create_flask_app" (lazy loading)
 * Environment: production
   WARNING: This is a development server. Do not use it in a production deployment.
   Use a production WSGI server instead.
 * Debug mode: on
 * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
```